lib/structuraid_core/designers/footings/centric_isolated.rb

```diff
@@ -17,8 +17,7 @@ class CentricIsolated
                  Steps::CheckBearingCapacity,
                  Steps::AssignAnalysisDirection,
                  Steps::CentricIsolated::ComputeRequiredRebarRatio,
-                 Steps::CentricIsolated::SetReinforcementLayersCoordinatesToFooting,
-                 Steps::CentricIsolated::SetInitialLongitudinalReinforcement,
+                 Steps::CentricIsolated::AddOrCheckReinforcement,
```

Comment:
We are delegating the responsibility of knowing which steps must be done, to `Steps::CentricIsolated::AddOrCheckReinforcement`